packages/db-ui/ezbackend-database-ui/src/Components/SchemaListItem.tsx

```diff
@@ -42,9 +42,9 @@ export default function SchemaListItem(props: ISchemaListItem) {
         <Typography variant="body2" noWrap>
           <Box
             component="div"
-            fontFamily={match ? 'monospace' : 'Inter'}
```

Comment:
Can we rename removeDbUI to removeDbUIFromString or something that makes more sense

Reply:
Also matching with what?

Reply:
Why not just make the function checkSelected()